Automatically deploy susi_server to Heroku with travis

Other FOSSASIA projects get deployed automatically to Heroku after pull requests. Please implement the same.

I have set up an app already here: https://susi-server.herokuapp.com

Some issue deploying the app. Please see below:

Unzipping /app/tmp/cache/.gradle/wrapper/dists/gradle-2.0-bin/5h57m9vra0mjv9qs45oqtsb5c0/gradle-2.0-bin.zip to /app/tmp/cache/.gradle/wrapper/dists/gradle-2.0-bin/5h57m9vra0mjv9qs45oqtsb5c0
       Set executable permissions for: /app/tmp/cache/.gradle/wrapper/dists/gradle-2.0-bin/5h57m9vra0mjv9qs45oqtsb5c0/gradle-2.0/bin/gradle
       
       FAILURE: Build failed with an exception.
       
       * What went wrong:
       Task 'stage' not found in root project 'fossasia-susi_server-771cb63'.
       
       * Try:
       Run gradlew tasks to get a list of available tasks. Run with --stacktrace option to get the stack trace. Run with --info or --debug option to get more log output.
       
       BUILD FAILED
       
       Total time: 6.933 secs
 !     ERROR: Failed to run Gradle!
       It looks like your project does not contain a 'stage' task, which Heroku needs in order
       to build your app. Our Dev Center article on preparing a Gradle application for Heroku
       describes how to create this task:
       https://devcenter.heroku.com/articles/deploying-gradle-apps-on-heroku
       
       If you're stilling having trouble, please submit a ticket so we can help:
       https://help.heroku.com
       
       Thanks,
       Heroku
 !     Push rejected, failed to compile Gradle app.
 !     Push failed

--- a/.travis.yml
+++ b/.travis.yml
@@ -15,3 +15,11 @@ install: true
 after_success:
 - bash <(curl -s https://codecov.io/bash)
 - codecov
+deploy:
+  provider: heroku
+  buildpack: https://github.com/loklak/heroku_buildpack_ant_loklak.git
+  api_key:
+    secure: "Secret key: travis encrypt $(heroku auth:token) --add deploy.api_key"
+  app: susi-server
+  on:
+    branch: development

